acceptance/chaos_test.go

```diff
@@ -113,27 +115,39 @@ func TestChaos(t *testing.T) {
 				return
 			default:
 			}
+			atomic.StoreInt64(&roundCopy, int64(round))
```

Comment:
just use atomics to update `round` directly (which can be a test-global variable).